pkg/controllers/dynakube/monitored_entities/reconciler.go

```diff
@@ -0,0 +1,69 @@
+package monitoredentities
+
+import (
+	"context"
+
+	"github.com/Dynatrace/dynatrace-operator/pkg/api/v1beta2/dynakube"
+	"github.com/Dynatrace/dynatrace-operator/pkg/clients/dynatrace"
+	"github.com/Dynatrace/dynatrace-operator/pkg/controllers"
+)
+
+type ReconcilerBuilder func(
+	dtClient dynatrace.Client,
+	dk *dynakube.DynaKube,
+	kubeSystemUUID string,
+) controllers.Reconciler
+
+func NewReconciler( //nolint
+	dtClient dynatrace.Client,
+	dk *dynakube.DynaKube,
+	kubeSystemUUID string,
+) controllers.Reconciler {
+	return &Reconciler{
+		dk:             dk,
+		dtClient:       dtClient,
+		kubeSystemUUID: kubeSystemUUID,
```

Comment:
the dynakube already has the `kubeSystemUUID` in its status, not necessary to pass it in separately

Reply:
done in https://github.com/Dynatrace/dynatrace-operator/pull/3447/commits/7e1e568e4e9045f69237947d9217d7ec22f8d380